nbformat/tests/test_sign.py

```diff
@@ -4,9 +4,11 @@
 # Distributed under the terms of the Modified BSD License.
 
 import copy
+import os
 import shutil
 import time
 import tempfile
+import testpath
```

Comment:
Sorry, I thought we already relied on testpath. You can add it to the test requirements in setup.py:

https://github.com/jupyter/notebook/blob/9bcac1d6293b376d4dedea94c36c7ad338c75771/setup.py#L160